code/modules/mob/living/simple_animal/revenant/revenant.dm

```diff
@@ -216,7 +216,7 @@
 	..()
 
 /datum/objective/revenant/check_completion()
-	if(!istype(owner.current, /mob/living/simple_animal/revenant) || !owner.current)
+	if(owner && !istype(owner.current, /mob/living/simple_animal/revenant) || !owner || !owner && !owner.current)
```

Comment:
Wouldn't that still runtime if owner is null when it does the istype check? It's an or check, so BYOND would presumably check both.